sidecar/nginx/lua/amalgam8.lua

```diff
@@ -498,6 +501,8 @@ function Amalgam8:update_state(input)
    local a8_actions = {}
    local err
 
+   self.myip = self:os_cmd("ifconfig eth0 | grep 'inet addr:' | cut -d: -f2 | awk '{ print $1}'", false)
```

Comment:
Move to init block? Doing this in update_state simply adds overhead.

Secondly, to get the local IP
```bash
hostname -i
```